### ISSUE TITLE
its not available on chrome extension store 

### ISSUE THREAD

**Issue by harrylongia** [jswanner/DontFuckWithPaste#115]
on the link it shows an error : 
404. That’s an error.
The requested URL was not found on this server. That’s all we know.

**Comment by kobitate** [comment]
For the time being, you can manually install the extension.

1. Return to [the main page of this repo](https://github.com/jswanner/DontFuckWithPaste/)
2. Click "Code" then "Download ZIP" (or clone the repo if you're feeling fancy)
3. Unpack the ZIP
4. Go to chrome://extensions/
5. Turn on Developer Mode in the top right, then click "Load unpacked"
6. Navigate to the root folder of the extension and Open

**Comment by jswanner** [comment]
@harrylongia, the Chrome Web Store took down the extension over the holidays because I hadn't filled out the "Privacy practices" details within their Developer Dashboard, which is something that was added since I last published an update to this extension. Once I returned home from my holiday travels, I filled out those details and I've been waiting for the Chrome Web Store to restore the extension. I only now noticed this issue, which prompted me to go back to check the Developer Dashboard, upon doing so I noticed something else was blocking the publishing of the extension. So, hopefully with resolution of that addition item the extension will be restored soon.
